inclusive-design/idrc
# Add svg tags around the logo svg

## Description

SVG tags around the IDRC logo svg was removed by mistake, which caused the logo on nav bar to disappear. Add back svg tags around the logo svg.

## Steps to test

1. Go to main page
2. Check that IDRC logo on top left corner is rendered properly
